src/Aquifer.Jobs/Program.cs

```diff
@@ -54,7 +55,8 @@
         services.AddSingleton<IAquiferApiManagementClient, AquiferApiManagementClient>();
         services.AddHttpClient<IIpAddressLookupHttpClient, IpAddressLookupHttpClient>()
             .AddPolicyHandler(HttpPolicyExtensions.HandleTransientHttpError()
-                .WaitAndRetryAsync(Backoff.DecorrelatedJitterBackoffV2(TimeSpan.FromSeconds(0.5), 2)));
+                .OrResult(res => res.StatusCode == HttpStatusCode.TooManyRequests)
+                .WaitAndRetryAsync(Backoff.DecorrelatedJitterBackoffV2(TimeSpan.FromSeconds(5), 2)));
```

Comment:
The only thing calling the IP service is an Azure function so a longer median wait timeout of 5 seconds seemed reasonable.